processor/spanmetricsprocessor/processor_test.go

```diff
@@ -392,22 +393,23 @@ func TestProcessorConsumeTraces(t *testing.T) {
 				return true
 			},
 			traces: []pdata.Traces{spanWithLargeTimestamp},
-			renames: []Rename{
-				{
-					Attributes: []Dimension{
-						{Name: "key_that_does_not_exist"},
-					},
-					NewCallsTotalMetricName: "this_should_not_match_calls_total",
-					NewLatencyMetricName:    "this_should_not_occur_latency",
-				},
-				{
-					Attributes: []Dimension{
-						{Name: stringAttrName},
-					},
-					NewCallsTotalMetricName: "new_name_calls_total",
-					NewLatencyMetricName:    "new_name_latency",
-				},
-			},
+			// TODO: CLAIRE: FIX THIS
```

Comment:
Does this test need to be removed or updated?

Reply:
ah oops, I missed a few TODOs 🤦 will fix those, thanks!